src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4570,4 +4597,168 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void TerminalPage::_onTabDragStarting(winrt::Microsoft::UI::Xaml::Controls::TabView sender,
+                                          winrt::Microsoft::UI::Xaml::Controls::TabViewTabDragStartingEventArgs e)
+    {
+        // Get the tab impl from this event.
+        const auto& eventTab = e.Tab();
```

Comment:
however, disprefer `const &` for locals; just use `const auto`

Reply:
wait really? I maybe I missed (or spaced) the discussion but I thought `const auto&` was still preferred?

Reply:
cause like, the plethora of `if (const auto& foo{ Bar() }) {}` we have

Reply:
Meh. I guess it's a new addition to the rules that I am not yet used to seeing. It looks like it can capture a temporary _just fine_ and extend its lifetime. Crazy.